public/styles/styles.scss

```diff
@@ -1,8 +1,10 @@
 @use "public/styles/globalVariables";
 @import 'fonts';
 
-*, body {
+html, body, div, button, p, input, .mat-checkbox, select, .mat-option, span, pre {
```

Comment:
Most likely you don't need overriding body elements, placing font-family just on the HTML or body itself should do its job as all other elements are their children so the same styles should apply to them as well unless they are tempered with in code directly by class or id.